<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="maskOne">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Maske</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inpaintOne">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inpaint</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageOne">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Eksempel 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="glattingOverskrift">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Inpainting</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>271</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Det legges på en maske på original-&lt;br/&gt;bildet som består av tre sorte &lt;br/&gt;firkanter. Deretter blir informasjonen&lt;br/&gt;i disse firkantene forsøkt gjenskapt&lt;br/&gt;ved hjelp av inpainting.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;Velg mellom tre eksempelbilder til&lt;br/&gt;høyre. Se masken som legges på, eller&lt;br/&gt;resultatet av inpaintingen.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;Lagre bildet ved å klikke på &lt;br/&gt;lagreknappen nede til venstre.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inpaintingCode">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vis kode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="maskTwo">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Maske</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageTwo">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Eksempel 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inpaintTwo">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inpaint</string>
   </property>
  </widget>
  <widget class="QPushButton" name="maskThree">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Maske</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageThree">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Eksempel 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inpaintThree">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inpaint</string>
   </property>
  </widget>
  <widget class="imagewidget" name="imagewidget" native="true">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>59</y>
     <width>431</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lagre</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>imagewidget</class>
   <extends>QWidget</extends>
   <header>imagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
